eng/pipelines/templates/jobs/perf.yml

```diff
@@ -35,6 +35,13 @@ extends:
     - template: /eng/pipelines/templates/steps/install-dotnet.yml
       parameters:
         EnableNuGetCache: false
+    - ${{ if parameters.Profile }}:
```

Comment:
This doesn't look like a declared parameter so I wouldn't expect this to work as is.

Reply:
It is declared here: https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/jobs/perf.yml#L59-L61

What do I need to do to pass it to this template?

Reply:
You will need to specify it as a parameter at the top of this file as well and then make sure it is passed through.

Reply:
Fixed